### PR TITLE
Add Basic UI & Run the Program 

### PR DESCRIPTION
**Pull request by andym80**
## GOOD REVIEWER ## 
Created an **app.rb** file that will serve as a  console app entry point. 
**Methods added.**

- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.
- 
**main.rb file** 
should do the following:
Present the user with a list of options to perform.
It Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

All suggestions are welcomed !

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    puts '7 - Exit'
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
+
+main
```
Comment (mateo951) [inline main.rb:68]: main.rb the entry point is defined, this is a method called main that is invoked at the end of the file ✔

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
+require_relative 'handlers'
+require './book_ui'
+require './person_ui'
+require './rental_ui'
+
+class App
```
Comment (mateo951) [inline main.rb:6]: Please be sure to create an app.rb file that will serve as their console app entry-point 🤓 You can check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point) if you need guidance.

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @andym80, 

**🔥 Highlights 🔥**
- UI 😎

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: main.rb (hunk 2 above)
Comment (andym80) [reply main.rb:6]: @mateo951  check on Handlers is included at  App method.

File: handlers.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.casecmp == 'y'
```
Comment (mmsesay) [inline handlers.rb]: - [ ] I got an error when I tried to create a student. I think it has to do with the case conversion.

| Error |
:-------:|
Screen Shot 2022-11-04 at 9 01 13 PM |



- [ ]  If you are trying to get the lowercase for the input please use `.downcase` 

Example: 
```rbuy
parent_permission = gets.chomp.downcase == "y"
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @andym80, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: handlers.rb (hunk 3 above)
Comment (andym80) [reply handlers.rb]: yes i followed rubocop and changed to **casecmp .** @mmsesay

**Review by AshakaE — APPROVED** [review]
Hi @andym80 , 

Thanks for making the changes 👏 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**